h/views/api_flags.py

```diff
@@ -22,6 +22,23 @@ def create(context, request):
     return HTTPNoContent()
 
 
+@api_config(route_name='api.flags',
+            request_method='GET',
+            link_name='flag.index',
+            description='List a users flagged annotations for review.',
+            effective_principals=security.Authenticated)
```

Comment:
Might it be friendly to have a version of this endpoint for logged-out users which just returns the empty list? At the moment `GET /api/flags` without authentication will return a 404.